SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift

```diff
@@ -65,11 +64,16 @@ class HomeViewController: UIViewController {
 
         let backButton = UIBarButtonItem(title: "Close", style: .plain, target: self, action: #selector(closeApp))
         self.navigationItem.leftBarButtonItem  = backButton
+        
     }
     @objc func closeApp() {
         exit(0)
 
     }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        self.removeSpinner()
```

Comment:
why here ?? this is arbitrary,  is not what the history saids ad if you do it in the view controller(which is clearly not what the story says ) at least put it when the service has a response